libs/external-db-firestore/src/firestore_schema_provider.ts

```diff
@@ -94,18 +125,26 @@ export default class SchemaProvider implements ISchemaProvider {
         })
     }
 
-    async describeCollection(collectionName: string): Promise<ResponseField[]> {
+    async changeColumnType(_collectionName: string, _column: InputField): Promise<void> {
+        throw new Error('Method not implemented.')
```

Comment:
In my opinion, this method is supported.
Because we can change the type of the column in the _descriptor table.
It will be the responsibilty of the user to change the content of the column according to the new coluimn type.